front/src/config/i18n/en.json

```diff
@@ -1208,6 +1257,25 @@
       "acceleration-z": "Acceleration Z",
       "bed-activity": "Bed Activity"
     },
+    "health": {
+      "battery": "Battery",
```

Comment:
Why do use the common `Battery` word?

Reply:
I change it by deviceFeatureCategory.battery.shortCategoryName